tests/lib.rs

```diff
@@ -312,18 +312,19 @@ fn proof_verifier() {
     }
 
     let pp = unsafe {
-        let buff = std::fs::read("tests/pub_params_dev.bin")
+        let buff = std::fs::read("temp_crs.bin")
```

Comment:
We should file a tech-debt to find a robust solution for this CRS generation problem

Reply:
I mean, the tests are not even passing now. I don't really see why.. 

We need for sure to find a better solution.